Asciidoctor error on Make - MacOS

Using MacOS and compiling the guide the following error shows up:
```
asciidoctor --failure-level WARNING -b html5 guide.adoc
asciidoctor: WARNING: chapters/extensions/VK_KHR_sampler_ycbcr_conversion.adoc: line 19: id assigned to section already in use: formats
make: *** [guide.html] Error 1
```
However, the guide.html file is generated. Is it ok?

--- a/.gitignore
+++ b/.gitignore
@@ -1,0 +1,2 @@
+# when building single page guide
+guide.html

--- a/README.adoc
+++ b/README.adoc
@@ -21,6 +21,11 @@ image::images/khronos_logo.png[Khronos logo]
 
 The Vulkan Guide is designed to help developers get up and going with the world of Vulkan. It is aimed to be a light read that leads to many other useful links depending on what a developer is looking for. All information is intended to help better fill the gaps about the many nuances of Vulkan.
 
+[NOTE]
+====
+The Vulkan Guide can be built as a single page using `asciidoctor guide.adoc`
+====
+
 :leveloffset: 1
 
 = Logistics Overview

--- a/chapters/depth.adoc
+++ b/chapters/depth.adoc
@@ -15,7 +15,7 @@ While stencil is closely related depth, this chapter does not aim to cover it ou
 ====
 
   *  <<graphics-pipeline, Graphics Pipeline>>
-  *  <<formats, Formats>>
+  *  <<depth-formats, Depth Formats>>
   *  <<depth-buffer-as-a-vkimage, Depth Buffer as a VkImage>>
   **  <<layout, Layout>>
   **  <<clearing, Clearing>>
@@ -45,8 +45,8 @@ The concept of "depth" is only used for xref:{chapters}what_vulkan_can_do.adoc[g
 
 Inside the `VkGraphicsPipelineCreateInfo` there are many different values related to `depth` that can be controlled. Some states are even xref:{chapters}dynamic_state.adoc[dynamic] as well.
 
-[[formats]]
-== Formats
+[[depth-formats]]
+== Depth Formats
 
 There are a few different depth formats and an implementation may expose support for in Vulkan.
 

--- a/chapters/extensions/VK_KHR_sampler_ycbcr_conversion.adoc
+++ b/chapters/extensions/VK_KHR_sampler_ycbcr_conversion.adoc
@@ -15,7 +15,7 @@ Promoted to core in Vulkan 1.1
 
 All the examples below use a `4:2:0` multi-planar {YCbCr} format for illustration purposes.
 
-[[formats]]
+[[multi-planar-formats]]
 == Multi-planar Formats
 
 To represent a {YCbCr} image for which the Y' (luma) data is stored in plane 0, the C~B~ blue chroma difference value ("U") data is stored in plane 1, and the C~R~ red chroma difference value ("V") data is stored in plane 2, an application would use the `VK_FORMAT_G8_B8_R8_3PLANE_420_UNORM format`.

--- a/chapters/formats.adoc
+++ b/chapters/formats.adoc
@@ -3,7 +3,7 @@
 
 ifndef::chapters[:chapters:]
 
-[[Formats]]
+[[formats]]
 = Formats
 
 Vulkan formats are used to describe how memory is laid out. This chapter aims to give a high-level overview of the variations of formats in Vulkan and some logistical information for how to use them. All details are already well specified in both the link:https://www.khronos.org/registry/vulkan/specs/1.2-extensions/html/vkspec.html#formats[Vulkan Spec format chapter] and the link:https://www.khronos.org/registry/DataFormat/specs/1.3/dataformat.1.3.html[Khronos Data Format Specification].
@@ -91,7 +91,7 @@ Some formats have both a depth and stencil component and can be accessed separat
 link:https://www.khronos.org/registry/vulkan/specs/1.2-extensions/man/html/VK_KHR_separate_depth_stencil_layouts.html[VK_KHR_separate_depth_stencil_layouts] and link:https://www.khronos.org/registry/vulkan/specs/1.2-extensions/man/html/VK_EXT_separate_stencil_usage.html[VK_EXT_separate_stencil_usage], which are both promoted to Vulkan 1.2, can be used to have finer control between the depth and stencil components.
 ====
 
-More information about depth format can also be found in the xref:{chapters}depth.adoc#formats[depth chapter].
+More information about depth format can also be found in the xref:{chapters}depth.adoc#depth-formats[depth chapter].
 
 === Compressed
 
@@ -111,7 +111,7 @@ representation of multiple pixels encoded interdependently within a region.
 
 === Planar
 
-link:https://www.khronos.org/registry/vulkan/specs/1.2-extensions/man/html/VK_KHR_sampler_ycbcr_conversion.html[VK_KHR_sampler_ycbcr_conversion] and link:https://www.khronos.org/registry/vulkan/specs/1.2-extensions/man/html/VK_EXT_ycbcr_2plane_444_formats.html[VK_EXT_ycbcr_2plane_444_formats] add xref:{chapters}VK_KHR_sampler_ycbcr_conversion.adoc#formats[multi-planar formats] to Vulkan. The planes can be accessed separately with `VK_IMAGE_ASPECT_PLANE_0_BIT`, `VK_IMAGE_ASPECT_PLANE_1_BIT`, and `VK_IMAGE_ASPECT_PLANE_2_BIT`.
+link:https://www.khronos.org/registry/vulkan/specs/1.2-extensions/man/html/VK_KHR_sampler_ycbcr_conversion.html[VK_KHR_sampler_ycbcr_conversion] and link:https://www.khronos.org/registry/vulkan/specs/1.2-extensions/man/html/VK_EXT_ycbcr_2plane_444_formats.html[VK_EXT_ycbcr_2plane_444_formats] add xref:{chapters}VK_KHR_sampler_ycbcr_conversion.adoc#multi-planar-formats[multi-planar formats] to Vulkan. The planes can be accessed separately with `VK_IMAGE_ASPECT_PLANE_0_BIT`, `VK_IMAGE_ASPECT_PLANE_1_BIT`, and `VK_IMAGE_ASPECT_PLANE_2_BIT`.
 
 === Packed
 

